scripts/doctests.py

```diff
@@ -33,11 +33,46 @@ def check_ipynb(dirn):
                     raise
                 else:
                     print(f'Ran notebook successfully.')
-    return 0
+
+def check_rstorm(dirn):
+    env = {**os.environ, 'SYN_LOG_LEVEL': 'DEBUG'}
+
+    for fdir, dirs, fns in os.walk(dirn):
+        if '.ipynb_checkpoints' in dirs:
+            dirs.remove('.ipynb_checkpoints')
+        if '_build' in dirs:
+            dirs.remove('_build')
+        for fn in fns:
+            if fn.endswith('.rstorm'):
+
+                oname = fn.rsplit('.', 1)[0]
+                oname = oname + '.rst'
+                sfile = os.path.join(fdir, fn)
+                ofile = os.path.join(fdir, oname)
+
+                args = ['python', '-m', 'synapse.tools.rstorm', '--save', ofile, sfile]
+
+                try:
+                    supb = subprocess.run(args, capture_output=True, timeout=60, check=True, env=env)
+                except Exception as e:
+                    raise
+                else:
+                    print(f'Ran {ofile} successfully.')
 
 def main():
     try:
-        return check_ipynb(docspath)
+        check_ipynb(docspath)
+    except subprocess.CalledProcessError as e:
+        print(f'Error executing notebook: {str(e)}')
+        print(f'Stdout:\n{e.stdout.decode()}')
+        print(f'Stderr:\n{e.stderr.decode()}')
+        return 1
+    except:
+        traceback.print_exc()
+        return 1
+
+    try:
+        check_rstorm(docspath)
     except subprocess.CalledProcessError as e:
         print(f'Error executing notebook: {str(e)}')
```

Comment:
Should this be "Error executing rstorm"?

Reply:
Fixed